lib/oppen/token.rb

```diff
@@ -35,6 +43,8 @@ def initialize(value, width: value.length)
         super()
       end
 
+      # Convert token to String.
+      #
       # @return [String]
       def to_s = value
     end
```

Comment:
Doc comments for `to_s` are usually not needed for ruby, because they're so evident and so used … But I don't mind it.